### PR TITLE
Adding LocalAuthentication RN docs

### PR DESCRIPTION
**Pull request by vivek-dhayalan**

### CODE REVIEW

**Review by prasanth-reddy-wavemaker-com — APPROVED** [review]

File: learn/react-native/device-biometric-authentication.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+![LocalAuthentication](/learn/assets/install-localauth.png)
+
+## How to list supported local authentication
+
+LocalAuthentication plugin provides an API `supportedAuthenticationTypeAsync()` to list supported authentication types available in a device.
+This API returns a promise that resolves to an array of strings, each of which is a constant defined in the [`AuthenticationType`](https://docs.expo.dev/versions/latest/sdk/local-authentication/#authenticationtype) enum.
+
+## How to invoke any local authentication
+
+'authenticateAsync()' API is used to invoke any local authentication. This API returns a promise that resolves to a boolean value indicating whether or not the authentication was successful.
```
Comment (swethakundaram) [inline learn/react-native/device-biometric-authentication.md:39]: 'authenticateAsync()' --> add single code block `authenticateAsync()`

File: learn/react-native/device-biometric-authentication.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+    Page.refresh();
+};
+```
+
+## How to invoke only Biometric authentication 
+
+We can restrict the authentication to only biometric authentication by passing the [`options`](https://docs.expo.dev/versions/latest/sdk/local-authentication/#localauthenticationauthenticateasyncoptions) parameter to the `authenticateAsync()` API.
+
+- `disableDeviceFallback` -- If set to true, the user will be unable to authenticate with biometrics (Touch ID or Face ID) and will be asked for their passcode instead. Defaults to false.
+- `requireConfirmation` -- Whether to show a dialog after successful biometric authentication. Defaults to true on Android.
```
Comment (swethakundaram) [inline learn/react-native/device-biometric-authentication.md:80]: :::note
`requireConfirmation` applies to Android only, and the default value sets to true. 
:::

File: learn/react-native/device-biometric-authentication.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
+---
+title: "Device Biometric Authentication"
+id: "device-biometric-authentication"
+sidebar_label: "Device Biometric Authentication"
```
Comment (swethakundaram) [inline learn/react-native/device-biometric-authentication.md:4]: sidebar_label: "Biometric Authentication"

File: learn/react-native/device-biometric-authentication.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+---
+Embracing biometric authentication in mobile apps, such as facial recognition and fingerprint scanning, 
+is an investment in robust security and user convenience. Biometric offers superior security, reducing the risk of breaches, and eliminating the hassle of remembering passwords. 
+A streamlined authentication process contributes to a seamless user experience, fostering trust and enhancing user retention. 
+While implementing, privacy concerns must be addressed responsibly. Despite these challenges, biometrics have redefined digital security standards in the mobile realm, 
+making it a preferred choice for many applications in our ever-evolving digital landscape.
+
+## Expo's LocalAuthentication Plugin
+
+Expo's [LocalAuthentication](https://docs.expo.io/versions/latest/sdk/local-authentication/) plugin provides an API to authenticate with the device's local authentication hardware, such as Touch ID, Face ID, or the Fingerprint API on Android.
```
[the remark this answers is not in the document]
Comment (vivek-dhayalan) [reply learn/react-native/device-biometric-authentication.md]: Updated

File: website/sidebars.json [hunk 5]
```diff
@@ -690,6 +690,13 @@
 						"react-native/web-view",				
 						"react-native/react-native-embed"
 					]
+				},
+				{
+					"type": "category",
+					"label": "Examples/Use-Case",
```
[the remark this answers is not in the document]
Comment (vivek-dhayalan) [reply website/sidebars.json]: Updated

File: learn/react-native/device-biometric-authentication.md [hunk 6]
```diff
@@ -0,0 +1,122 @@
+---
+title: "Device Biometric Authentication"
```
[the remark this answers is not in the document]
Comment (vivek-dhayalan) [reply learn/react-native/device-biometric-authentication.md]: updated

File: learn/react-native/device-biometric-authentication.md (hunk 1 above)
Comment (vivek-dhayalan) [reply learn/react-native/device-biometric-authentication.md]: updated

File: learn/react-native/device-biometric-authentication.md (hunk 2 above)
Comment (vivek-dhayalan) [reply learn/react-native/device-biometric-authentication.md]: updated

File: learn/react-native/device-biometric-authentication.md (hunk 3 above)
Comment (vivek-dhayalan) [reply learn/react-native/device-biometric-authentication.md]: Can we have different title and side bar label?

**Review by swethakundaram — APPROVED** [review]

**Pull request merged**